app/routes/review.py

```diff
@@ -181,20 +183,7 @@ def add_review():
   db.session.commit()
 
   if request.json.get('include_wallet'):
-    request_url = f'{wally_api_base}/wallets/create'
-    headers = {
-      'Authorization': f'Bearer {wally_auth_token}',
-      'Content-Type': 'application/json',
-    }
-
-    payload = {
-      'id': f'user_{str(user.id)}',
-      'email': user.email, # owner is the current user, so owner email is current user email
-      'tags': ['user']
-    }
-
-    response = requests.post(request_url, headers=headers, json=payload)
-    response.raise_for_status()
+    data = create_wallet(user=user)
```

Comment:
should we just move this down to the other if statement? that way all the wallet logic is in one place?